components/SingleEntry/SingleEntry.style.js

```diff
@@ -0,0 +1,6 @@
+import { styled } from "styled-components";
+
+export const EntryContent = styled.div`
+  margin: 0 10px;
+  margin-top: 90px;
+`;
```

Comment:
- Please try to use a semantic HTML element instead of the generic div element.
- There are two separate margin declarations. The shorthand property for setting all sides of the margin could be used here for simplicity and readability.

```suggestion
export const EntryContent = styled.div`
   margin: 90px 10px 0;
`;
```

Reply:
I tried to make the semantic more selfexplaining